README.md

```diff
@@ -1,3 +1,18 @@
+# 💀 NOTICE - This Process is Deprecated! 💀
+**The awslogin python script is no longer in use at BYU.**
+
+The new AWS login page is <https://aws.byu.edu>. 
+
+# Approved 'awslogin' Alternatives
+To login from the command line, setup the __AWS CLI__
+  - Start at <https://aws.byu.edu>
+  - Click the account name you want to login to
+  - Select `Command Line or Programmatic Access` and follow the directions on how to update your local aws config files. 
```

Comment:
I don't see any instructions here about setting up your configuration file. Maybe put something like this:

For each AWS account that exists you'll need to create a profile configuration in your `.aws/config` file. Each configuration should look like this, replacing `<AWS_ACCOUNT_NAME>` with the actual account name and `<AWS_ACCOUNT_ID>` with the actual account id.

```
[profile <AWS_ACCOUNT_NAME>]
sso_start_url = https://byulogin.awsapps.com/start
sso_region = us-west-2
sso_account_id = <AWS_ACCOUNT_ID>
sso_role_name = PowerUser-<AWS_ACCOUNT_ID>
region = us-west-2
output = json
```